docs/pages/database-access/auto-user-provisioning/aws-redshift.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: AWS Redshift Automatic User Provisioning 
+description: Configure automatic user provisioning for AWS Redshift.
+---
+
+(!docs/pages/includes/database-access/auto-user-provisioning/intro.mdx!)
+
+## Prerequisites
+
+- Teleport cluster v14.1.3 or higher with a configured [AWS
```

Comment:
Manually applied #34413 and updated the actual release version [v14.1.3](https://github.com/gravitational/teleport/releases/tag/v14.1.3).

Will update the release number on master later.